src/Bicep.LangServer/Snippets/Templates/res-virtual-wan.bicep

```diff
@@ -0,0 +1,12 @@
+// Virtual WAN
+resource ${1:virtualWan} 'Microsoft.Network/virtualWans@2020-07-01' = {
+  name: ${2:'name'}
+  location: resourceGroup().location
+  properties: {
+    type: '${3|Standard,Basic|}'

```

Comment:
Move quotes from outside to inside each string e.g. `type: ${3|'Standard','Basic'|}`